### DISCUSSION TITLE
transmission-rss won't find the config

### DISCUSSION THREAD

**Discussion by ghost** [haugene/docker-transmission-openvpn#2067] [Q&A] [answered]
### Is there a pinned issue for this?

- [X] I have read the pinned issues and could not find my issue

### Is there an existing or similar issue/discussion for this?

- [X] I have searched the existing issues
- [X] I have searched the existing discussions

### Is there any comment in the documentation for this?

- [X] I have read the documentation, especially the FAQ and Troubleshooting parts

### Is this related to a provider?

- [X] I have checked the provider repo for issues
- [X] My issue is NOT related to a provider

### Are you using the latest release?

- [X] I am using the latest release

### Have you tried using the dev branch latest?

- [X] I have tried using dev branch

### Config used

version: '3.9'
services:
    transmission-openvpn:
        image: haugene/transmission-openvpn
        hostname: transmission
        container_name: transmission-openvpn
        cap_add:
            - NET_ADMIN
        volumes:
            - '/home/hulk/thebox/:/data'
        environment:
            - OPENVPN_PROVIDER=null
            - OPENVPN_CONFIG=null
            - OPENVPN_USERNAME=null
            - OPENVPN_PASSWORD=null
            - LOCAL_NETWORK=192.168.0.0/16
        logging:
            driver: json-file
            options:
                max-size: 10m
        ports:
            - '9091:9091'
    transmission-rss:
        image: haugene/transmission-rss
        hostname: transmission-rss
        container_name: transmission-rss
        depends_on:
            - transmission-openvpn
        volumes:
            - '/home/hulk/thebox/:/etc/transmission-rss.conf'

transmission-rss.conf

feeds:
  - url: 
    download_path: /home/hulk/thebox/

server:
  host: ip
  port: 9091
  rpc_path: /transmission/rpc

### Current Behavior

the transmission and openvpn work great. transmission-rss just wont find the config file.

### Expected Behavior

transmission-rss to read the config file 

### How have you tried to solve the problem?

tried to have the config in /etc/. it wont work either.     

docker-compose
environment:
            - RSS_URL=<snip> works but i want it to read the config file

### Log output

[_transmission-rss_logs.txt](https://github.com/haugene/docker-transmission-openvpn/files/7559852/_transmission-rss_logs.txt)

### Environment

```markdown
- OS: ubuntu 20.04
- Docker: 20.10.7
```

### Anything else?

https://haugene.github.io/docker-transmission-openvpn/rss-plugin/
https://github.com/haugene/docker-transmission-openvpn/issues/1668

**Comment by pkishino** [comment, marked as answer]
this:
```- '/home/hulk/thebox/:/etc/transmission-rss.conf'```
does not make sense..you are trying to map a folder to a file.. 
you should try 
```- '/home/hulk/thebox/transmission-rss.conf:/etc/transmission-rss.conf'```

this is not really a bug I believe but a config problem, hence moving to discussions

**Reply by ghost** [reply]
thanks it works now
